### ISSUE TITLE
External scripts without the necessary `Access-Control-Allow-Origin` header are rejected in karma v1.1.1

### ISSUE THREAD

**Issue by dotch** [karma-runner/karma#2249]
### Expected behavior

External scripts can be loaded regardless of the `Access-Control-Allow-Origin` headers.
### Actual behavior

External scripts without the necessary `Access-Control-Allow-Origin` header are rejected with:

```
Script from origin '...' has been blocked from loading by 
Cross-Origin Resource Sharing policy: 
No 'Access-Control-Allow-Origin' header is present on the 
requested resource. Origin '...' is therefore not allowed access.
```

This happens with Karma v1.1.1 when external scripts are served without the necessary `Access-Control-Allow-Origin` header. 
In Karma v1.1.0 the scripts were loaded. This change in behavior is due to https://github.com/karma-runner/karma/commit/5690ffe82bb7ff195576f6f43dd8c281e51aa833

Sometimes users might not have control over the external scripts they need to include in their test run. An option to turn off the new default behavior might be sensible.
### Enviroment Details
- Karma version (output of `karma --version`): v1.1.1
- A modern browser that respects `crossorigin="anonymous"`

**Comment by frnk94** [comment]
Since: https://github.com/karma-runner/karma/pull/2219

**Comment by maksimr** [comment]
Already fixed in PR #2318
